### PR TITLE
Fare profile enhancement zone in series

### PR DESCRIPTION
**Pull request by nick-knowles**
This is a proposed minor change tarising out of Fare profile examples - allows a routing constraint of a specified sequence of TARIFF ZONEs to be specified as ZONES IN SEQUENCE associated with a SERIES CONSTRAINt associated with DISTANCE MATRIX defining a point to point fare between  a start and end zone

### CODE REVIEW

File: xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd [hunk 1]
```diff
@@ -328,5 +333,72 @@ Rail transport, Roads and Road transport
 			<xsd:element ref="SeriesConstraintRef" minOccurs="0"/>
 		</xsd:sequence>
 	</xsd:group>
+	<!-- == SZONE IN SERIES================================ -->
```
Comment (skinkie) [inline xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd:336]:
```suggestion
	<!-- == ZONE IN SERIES================================ -->
```

**Review by skinkie — APPROVED** [review]

**Review by Aurige — APPROVED** [review]

File: xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd [hunk 2]
```diff
@@ -328,5 +335,72 @@ Rail transport, Roads and Road transport
...
+	<xsd:complexType name="ZoneInSeries_VersionedChildStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a ZONE IN SERIES.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="VersionedChildStructure">
+				<xsd:sequence>
+					<xsd:group ref="ZoneInSeriesGroup"/>
+				</xsd:sequence>
+				<xsd:attribute name="order" type="xsd:integer"/>
```
Comment (skinkie) [inline xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd:388]: @Aurige this should be a positiveInteger?

Comment (skinkie) [inline xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd:388]:
```suggestion
				<xsd:attribute name="order" type="xsd:positiveInteger"/>
```

File: xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd [hunk 3]
```diff
@@ -328,5 +335,72 @@ Rail transport, Roads and Road transport
...
+							<xsd:group ref="EntityInVersionGroup" minOccurs="0"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="VersionedChildGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="ZoneInSeriesGroup"/>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="ZoneInSeriesIdType"/>
```
Comment (skinkie) [inline xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd:374]:
```suggestion
					<xsd:attribute name="id" type="ZoneInSeriesIdType" use="required"/>
```

File: xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd (hunk 2 above)
Comment (Aurige) [reply xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd:388]: that's right, this is the new rule ...

File: xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd [hunk 4]
```diff
@@ -328,5 +335,72 @@ Rail transport, Roads and Road transport
...
+	</xsd:complexType>
+	<xsd:group name="ZoneInSeriesGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for a ZONE IN SERIES.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element ref="TariffZoneRef_"/>
+			<xsd:element name="InterchangeAllowed" type="xsd:boolean" default="true" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether interchaneg is allowed in a zone</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd:400]:
```suggestion
					<xsd:documentation>Whether interchange is allowed in a zone.</xsd:documentation>
```

File: xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd [hunk 5]
```diff
@@ -328,5 +335,72 @@ Rail transport, Roads and Road transport
...
+	<xsd:complexType name="zonesInSeries_RelStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a list of ZONEs IN SERIES.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="strictContainmentAggregationStructure">
+				<xsd:sequence>
+					<xsd:element name="ZoneInSeries" type="ZoneInSeries_VersionedChildStructure" maxOccurs="unbounded">
+						<xsd:annotation>
+							<xsd:documentation>An orderd sequence of ZONEs IN SEQUENCE constrainign the permitted routing.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd:348]:
```suggestion
							<xsd:documentation>An orderd sequence of ZONEs IN SEQUENCE constraining the permitted routing.</xsd:documentation>
```

File: xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd [hunk 6]
```diff
@@ -16,9 +16,11 @@
 				<Date>
 					<Created>2010-09-04</Created>
 				</Date>
-				<Date>
-					<Modified>2011-02-05</Modified>
-					<!-- Name Space changes -->
+				<Date><Modified>2011-02-05</Modified>Name Space changes 
+				</Date>
+				<Date><Modified>2023-12-09</Modified>Add ZOne in Series
```
Comment (trurlurl) [inline xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd:21]:
```suggestion
				<Date><Modified>2023-12-09</Modified>Add Zone in Series 
```

File: xsd/netex_part_3/part3_fares/netex_fareSeries_support.xsd [hunk 7]
```diff
@@ -44,20 +44,20 @@
...
+					</Category>
 					<Project>CEN TC278 WG3 SG9.</Project>
 				</Subject>
-				<Title>NeTEx   SERIES CONSTRAINT identifier types.</Title>
+				<Title>NeTEx SERIES CONSTRAINT identifier types.</Title>
 				<Type>Standard</Type>
 			</Metadata>
 		</xsd:appinfo>
-		<xsd:documentation>NetEx   SERIES CONSTRAINT identifeer types.</xsd:documentation>
+		<xsd:documentation>NetEx SERIES CONSTRAINT identifeer types.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_3/part3_fares/netex_fareSeries_support.xsd:60]:
```suggestion
		<xsd:documentation>NetEx SERIES CONSTRAINT identifier types.</xsd:documentation>
```

**Review by ue71603 — APPROVED** [review]

**Comment by hegedreiem** [comment]
Would it be useful to add something like a **zoneToZone** entry to the SeriesTypeEnumeration, to make it clear that the series constraint is based on zones?

**Review by Aurige — APPROVED** [review]

File: xsd/netex_part_3/part3_fares/netex_fareSeries_support.xsd [hunk 8]
```diff
@@ -44,20 +44,20 @@
...
+					</Category>
 					<Project>CEN TC278 WG3 SG9.</Project>
 				</Subject>
-				<Title>NeTEx   SERIES CONSTRAINT identifier types.</Title>
+				<Title>NeTEx SERIES CONSTRAINT identifier types.</Title>
 				<Type>Standard</Type>
 			</Metadata>
 		</xsd:appinfo>
-		<xsd:documentation>NetEx   SERIES CONSTRAINT identifeer types.</xsd:documentation>
+		<xsd:documentation>NetEx SERIES CONSTRAINT identifier types.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_3/part3_fares/netex_fareSeries_support.xsd:60]:
```suggestion
		<xsd:documentation>NeTEx SERIES CONSTRAINT identifier types.</xsd:documentation>
```

File: xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd [hunk 9]
```diff
@@ -328,5 +335,72 @@ Rail transport, Roads and Road transport
...
+						</xsd:annotation>
+					</xsd:element>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:element name="ZoneInSeries" substitutionGroup="VersionedChild">
+		<xsd:annotation>
+			<xsd:documentation>Use of a ZONE in sequence within a SERIES CONSTRAINT, for example to indicate that a series must route through a given sequence of TARIFF ZONEs, +v2.0
+</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd:358]:
```suggestion
			<xsd:documentation>Use of a ZONE in sequence within a SERIES CONSTRAINT, for example to indicate that a series must route through a given sequence of TARIFF ZONEs, +v2.0</xsd:documentation>
```

File: xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd [hunk 10]
```diff
@@ -328,5 +335,72 @@ Rail transport, Roads and Road transport
...
+	<xsd:complexType name="zonesInSeries_RelStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a list of ZONEs IN SERIES.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="strictContainmentAggregationStructure">
+				<xsd:sequence>
+					<xsd:element name="ZoneInSeries" type="ZoneInSeries_VersionedChildStructure" maxOccurs="unbounded">
+						<xsd:annotation>
+							<xsd:documentation>An orderd sequence of ZONEs IN SEQUENCE constraining the permitted routing.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd:348]:
```suggestion
							<xsd:documentation>An ordered sequence of ZONEs IN SEQUENCE constraining the permitted routing.</xsd:documentation>
```

**Comment by nick-knowles** [comment]
UML added
image

**Comment by nick-knowles** [comment]
TM UM:
image

**Comment by Aurige** [comment]
Part 3 doc updated

**Pull request merged**